src/api/spec/factories/staging_workflow.rb

```diff
@@ -0,0 +1,17 @@
+FactoryBot.define do
+  factory :staging_workflow do
+    project { nil }
```

Comment:
Factories are supposed to work. You require project.

Reply:
Removed.